public/javascript/components/surveyController.js

```diff
@@ -0,0 +1,52 @@
+/**
+ * Survey Controller
+ */
+
+import $ from 'jquery';
+import firebase from "firebase/app";
+import "firebase/database";
+
+// Submit button on click
+$('.survey__submit').click(function(e) {
+  let getCheckedButtons = getAllRadioButtonsChecked()
+  
+  e.preventDefault()
+
+  if (checkAllButtonsClicked()) {
+    writeDataIntoFirebase(getCheckedButtons)
```

Comment:
Wait, you sure we can't do?
```javascript
writeDataIntoFirebase(getAllRadioButtonsChecked())
```
Since we're returning the value in `getAllRadioButtonsChecked()`, it should evaluate to the final object that we're expecting.

Reply:
👍🏾